Week2task2/src/Main.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+public class Main {
+
+	public Main() {
+		// TODO Auto-generated constructor stub
+	}
+
+	public static void main(String[] args) {
+		Name person = new Name();
+		person.name = "Anna";
+		person.surname = "Wick";
+		Name person1 = new Name(); 
+		person1.name = "Justyna";
+		person1.surname = "Garyga";
+		Name person2 = new Name(); 
+		person2.name = "Anna";
+		person2.surname = "Nowak";
+		
+		
+		// create ArrayList, sort and print sorted list
+		List<Name> people = new ArrayList<>();
+		people.add(person);
+		people.add(person1);
+		people.add(person2);
+	
+		Arrays.sort(people);
```

Comment:
You will have to use Collections.sort. In order to this to work you will have to implement the interface Comparable in your Name class -> class Name implements Comparable and change the param that is received by compareTo method to Object person1. Then you'll have to cast this person1 into a Name -> Name p = (Name) person1.